tests/conftest.py

```diff
@@ -91,6 +92,7 @@ def init_db(qapp, qtbot, tmpdir_factory):
     source_dir.save()
 
     qapp.main_window.deleteLater()
+    signals.post_save.disconnect(receiver=qapp.main_window.miscTab.on_setting_update, sender=SettingsModel)
```

Comment:
Lets swap those lines around. And remove the `sender` parameter. I don't know why the test fails so I want to eliminate possible sources.

Reply:
Hi! A question, after removing the sender parameter it will consider it as `sender=any` right ? as the value is not provided

Reply:
The docs don't mention a sender parameter for `disconnect`.

Reply:
Works Now! Thanks @real-yfprojects and one of my friend @TheSpeedX

Reply:
That does work. However we now changed working code just for tests to pass that were failing due to a `peewee` bug.

Reply:
> Lets swap those lines around. And remove the `sender` parameter. I don't know why the test fails so I want to eliminate possible sources.

Hi! I actually tried this but it was still failing, any workaround suggested ?

Reply:
I found the underlying issue: See coleifer/peewee#2687.

> any workaround suggested?

We could leave the disconnect in place and reconnect `setup_autostart` for now.

Reply:
Like so

```suggestion
    # disconnects all signals because of bug coleifer/peewee#2687
    signals.post_save.disconnect(receiver=qapp.main_window.miscTab.on_setting_update, sender=SettingsModel)
    signals.post_save.connect(setup_autostart, sender=SettingsModel)  # workaround
    
    qapp.main_window.deleteLater()
```

Reply:
Hi! I had to add a disconnect signal for the autostart before the connect signal because the above code was failing since the autostart is already connected